ci/hack/rosa_hcp_setup.sh

```diff
@@ -4,20 +4,23 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
```

Comment:
This should be removed, that's what the `--debug` option is there for.

Reply:
fixed